code/controllers/subsystem/objects.dm

```diff
@@ -16,6 +16,7 @@ var/list/processing_objects = list()
 	NEW_SS_GLOBAL(SSobj)
 
 /datum/subsystem/obj/Initialize()
+	set background=1
```

Comment:
what purpose is this ssobj init change serving?

Reply:
This proc always gets sent to the background regardless on server init, causing a message log to pop in Dream Daemon (which makes sense given the amount of objects). This line just makes it so it does so right away and doesn't send the message. 
![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/d1949148-1ca0-4c73-b57b-41ee6504eec3)